websites/R/Roblox Forum/presence.ts

```diff
@@ -25,8 +30,20 @@ presence.on("UpdateData", async () => {
 		presenceData.state = `Profile: ${document
 			.querySelector("meta[property='og:title']")
 			.getAttribute("content")}`;
-	} else if (document.location.pathname.startsWith("/whats-new/"))
-		presenceData.details = "Browsing latest content";
+		presenceData.buttons = [
+			{
+				label: "View profile",
+				url: document.location.href,
+			},
+		];
+	} else if (document.location.pathname.startsWith("/admin.php"))
```

Comment:
Combine this else if with the others by using an OR operator

Reply:
i tried using || but i couldn't get that to work, I tried using && but only works with the first thing that starts

Reply:
nvm on that, i removed startsWith and it worked. i have no idea if this is the most efficient way of doing this, so lmk what u think

Reply:
yeah this deff isnt the way how I should do it, it errors on the checks -- but I have no idea what/or how I could fix this?

![image](https://user-images.githubusercontent.com/54631194/179148020-b9674f14-3ee5-4698-afc2-15e5286dcf49.png)

Reply:
![image](https://user-images.githubusercontent.com/54631194/179149511-b92f1c80-6797-437d-8bc6-bdbccb2b674b.png)
I have this locally, but only the first one works. The other 2 don't, don't know why

Reply:
tbf the amin.php one isn't needed, so i removed it

Reply:
![image](https://user-images.githubusercontent.com/54631194/179156343-c787c7c7-c89f-4c89-9e20-a0b43030591d.png)
if i use 1 = it works both, but it causes to refresh the page infinitely. if u use 2 == only 1 works

Reply:
i'm stuck on this one, id you could give me a push in the right direction i'd appreciate it a lot

Reply:
(also using startsWith doesn't work at all)

Reply:
I’m on holiday so this is hard to type out syntactically 

if(pathname.includes(“admin”) || pathname.includes(“approval-queue”) presenceData.details = “Moderation shenanigans”